Metro-Records/la-metro-councilmatic
# Debug unclosed link, remove AB361 banner

## Overview

This removes the outdated "virtual meetings" banner and closes a rogue open anchor tag causing styling bugs.

Closes #945 
Closes #946

## Testing Instructions

 * Make sure banner is gone from all relevant pages (index, event, events, and minutes)
 * Check that the non-link text under "Recent Activity" and "Board of Directors Meetings" sections of the `board-members` page are no longer styled as links
